integrations/utils/commonUtils.js

```diff
@@ -1,5 +1,4 @@
 import _ from "lodash";
-import logger from "../../utils/logUtil";
```

Comment:
Is this not used anywhere in this file?

Reply:
@ItsSudip , no we are not using it anywhere in file